feat: uniformize inputs and outputs for all algos (#179)
Substra/substra-documentation: substrafl_examples/strategies_examples/assets/dataset/description.md

The opener exposes 6 methods:

> - `get_data` returns a torch.FloatTensor containing the images and the labels in a dict

Reviewer: ```suggestion
- `get_data` returns a dictionary containing containing the images and the labels as numpy arrays
```

Author: done

Author: done
